### PR TITLE
Problem: No infrastruture for running pacemaker checks

### PR DESCRIPTION
**Pull request by knekrasov**
Solution:
1. Implement a 'client' for pacemaker-related operations.
2. Implement basic check:
```
./validate.py pacemaker --help
usage: CORTX Validator command pacemaker [-h] {configured} ...

positional arguments:
  {configured}
    configured  Basic sanity check of Pacemaker

optional arguments:
  -h, --help    show this help message and exit
```

### CODE REVIEW

File: py-utils/src/utils/validator/commands.py [hunk 1]
```diff
@@ -142,11 +156,47 @@ class ElasticsearchVCommand(VCommand):
...
 
         self._elasticsearch = ElasticsearchV()
 
     def process(self):
         """Validate elasticsearch status."""
 
         self._elasticsearch.validate(self.v_type, self.args)
+
+
+class PacemakerCommand(VCommand):
```
Comment (ujjwalpl) [inline py-utils/src/utils/validator/commands.py:169]: Please rename to PacemakerVCommand to match the conventions or we will need to change all the other classes to match

File: py-utils/src/utils/validator/commands.py [hunk 2]
```diff
@@ -16,12 +16,23 @@
 # For any questions about this software or licensing,
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 
-import errno
+import argparse
+
+from .pacemaker import PacemakerHealth
```
Comment (ujjwalpl) [inline py-utils/src/utils/validator/commands.py:21]: Please move this import to PacemakerVCommand (where required) as followed in the code.

**Review by ujjwalpl — CHANGES_REQUESTED** [review]

File: py-utils/src/utils/validator/commands.py [hunk 3]
```diff
@@ -34,9 +45,12 @@ def args(self):
     def v_type(self):
         return self._v_type
 
-    @staticmethod
-    def add_args(parser, cmd_class, cmd_name):
+    @classmethod
+    def add_args(cls, parser):
```
Comment (ujjwalpl) [inline py-utils/src/utils/validator/commands.py:49]: This will break other classes that make use of the old interface.

**Review by ujjwalpl — CHANGES_REQUESTED** [review]

File: py-utils/src/utils/validator/pacemaker.py [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+import logging
+import re
+from errno import ERANGE
+from subprocess import PIPE, Popen  # nosec
+from typing import List, Optional
+
+from .error import VError
+
+
+class CliExecutor:
```
Comment (ujjwalpl) [inline py-utils/src/utils/validator/pacemaker.py:26]: There is already a class called SimpleProcess. Please use the same as all the other classes are using.

**Review by ujjwalpl — CHANGES_REQUESTED** [review]

File: py-utils/src/utils/validator/pacemaker.py [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+        logging.debug('Issuing CLI command: %s', cmd)
+        out, err = process.communicate()
+        exit_code = process.returncode
+        logging.debug('Finished. Exit code: %d', exit_code)
+        if exit_code:
+            raise VError(exit_code, err)
+        return out
+
+
+class PacemakerHealth:
```
Comment (ujjwalpl) [inline py-utils/src/utils/validator/pacemaker.py:56]: Please rename to PacemakerV, again as per conventions.

**Review by ujjwalpl — CHANGES_REQUESTED** [review]

**Comment by knekrasov** [comment]
Closed in favor of https://github.com/Seagate/cortx-utils/pull/87 and https://github.com/Seagate/cortx-utils/pull/85

**Pull request closed without merging**